lib/diagnostic.js

```diff
@@ -39,17 +56,27 @@ const any = function (/* array, callback */) {
 // the callback returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.filter`
-const select = function (/* array, callback */) {
+const select = function (array, callback) {
+  let index = array.filter(callback);
+    if (index < 5){
+      return false;
+    }
 
+    return true;
 };
 
 // Question 5
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 // you may use any feature of JavaScript
 // other than `Array.prototype.find`
-const first = function (/* array, callback */) {
+const first = function (array, callback) {
```

Comment:
This function should return the first element that it finds, not a boolean value.